src/UnitTests/Core/Serialization/Json/JsonProjectSerializerTests.cs

```diff
@@ -32,7 +32,7 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace Reko.UnitTests.Analysis
+namespace Reko.UnitTests.Decompiler.Analysis
```

Comment:
Maybe should be `Reko.UnitTests.Core.Serialization.Json`

Reply:
Good catch!